legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -411,6 +411,11 @@ def _get_filing_types(filing_json: dict):
                     'priority': False if filing_type == 'annualReport' else priority_flag,
                     'waiveFees': filing_json['filing']['header'].get('waiveFees', False)
                 })
+            elif k == 'incorporationApplication':
+                filing_types.append({
+                    'filingTypeCode': Filing.FILINGS[k].get('code'),
+                    'futureEffective': ListFilingResource._is_future_effective_filing(filing_json)
```

Comment:
Should the future effective flag be added to the other filing types too? Also should incorporationApplication include the priority + waiveFees options too?

Reply:
By default the value is false. We need to add that only if it has a fee related to future effective.
Also, we dont have a priority + waive fee option for incorporation. I think we can add that if that feature is supported